### PR TITLE
Personal airbags now prevent fall damage

### PR DESCRIPTION
**Pull request by SECBATON-GRIFFON**
[content]

## What this does
makes the airbags open up on landing if the gravity that causes fall damage is greater than 0.5 gs (suicide jumps are exempt)

## Why it's good
additional use for this item that should've made sense

## Changelog
:cl:
 * rscadd: Personal airbags now prevent fall damage, if possible.

### CODE REVIEW

File: vgstation13.dme [hunk 1]
```diff
@@ -2859,7 +2859,7 @@
 #include "maprendering\maprendering.dm"
 #include "maps\_map.dm"
 #include "maps\_map_override.dm"
-#include "maps\tgstation.dm"
+#include "maps\test_multiz.dm"
```
Comment (DorainGrey) [inline vgstation13.dme]: This will cause issues.

Comment (SECBATON-GRIFFON) [reply vgstation13.dme]: i always do this, fixed

**Comment by Eneocho** [comment]
I can see it. Multi Z snaxi, the clown is running away from the HoS after melting all the guns. The officers want blood but the clown barricaded himself in the rooftop. One of the officers brought breaching tools, it's only time before the clown is brained and his brain turned into cake.
The clown pleads for mercy but is ignored. With a wistful smile and closed eyes, he says his final words on the radio as he jumps... "One day, while andy...".
Time slows down, clown can see his family, his wife, children. He can see the Honkmother and his rival the mime. The ground is closing in-
The clown opens his eyes. He's alive? A deflated airbag around him, coming from his stolen bag of holding. Saved by the bell, but there's much to do.

File: code/modules/multiz/fall_hit.dm [hunk 2]
```diff
@@ -36,7 +36,10 @@
 /mob/living/fall_impact(var/turf/landing)
 	var/gravity = get_gravity()
 	var/total_brute_loss = 0
-	if(gravity > 0.5)
+	var/obj/item/airbag/airbag = locate() in contents
+	if(mind && mind.suiciding)
+		airbag = null
```
Comment (Eneocho) [inline code/modules/multiz/fall_hit.dm]: Not liking this. How does the airbag (automatic) know this jump is suicidal in nature? Make it so it opens anyway, but you die of a heart attack during the fall.

Comment (Kurfursten) [reply code/modules/multiz/fall_hit.dm]: instead of doing a costly locate then nulling it, just don't locate it if you don't need it

Comment (SECBATON-GRIFFON) [reply code/modules/multiz/fall_hit.dm]: fixed it

**Pull request merged**